pkg/kubelet/apis/kubelet.config.k8s.io/v1beta1/defaults.go

```diff
@@ -49,11 +49,32 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 	if obj.ConfigTrialDuration == nil {
 		obj.ConfigTrialDuration = &metav1.Duration{Duration: 10 * time.Minute}
 	}
+	if obj.SyncFrequency == zeroDuration {
```

Comment:
Where does all these things come from? Net-new or copied from somewhere else?
Do we have unit tests for this defaulting anywhere?

Reply:
I went through this file again to refresh my memory, here are the changes:
- I sorted the defaults to the same field order as the config struct. 
- I swapped `boolVar(b)` for `utilpointer.BoolPtr(b)`.
- I swapped `temp := int32(i); obj.Field = &temp` for `utilpointer.Int32Ptr(i)`
- Defaults should not depend on other values in the object (`CPUManagerReconcilePeriod` was doing this).
- Maintained the correct default for `FailSwapOn`.

What kind of unit testing? I'm not sure a test that just checks whether defaults were applied is very useful; it's effectively just a checksum that you have to change every time you add a field.